Plugins/ConnectMocksPlugin/ConnectMockGenerator.swift

```diff
@@ -219,4 +224,28 @@ private extension MethodDescriptor {
             """
         }
     }
+
+    func callbackAvailabilityAnnotation() -> String? {
+        if self.options.deprecated {
+            // swiftlint:disable line_length
```

Comment:
@rebello95 @jhump Let me know if this is acceptable. I use SwiftLint in my own repos as well, so I know the rules exist for a purpose; however, I may propose disabling this specific rule in this specific context to improve legibility in the editor and to more closely mimic what will actually be output at the time of generation.

Reply:
This seems fine to me, doing additional gymnastics would be a bit excessive 😉